test/acceptance/cluster_api_auth/cluster_api_auth_test.go

```diff
@@ -26,11 +26,12 @@ import (
 )
 
 func TestClusterAPIAuth(t *testing.T) {
+	t.Setenv("TEST_WEAVIATE_IMAGE", "weaviate/test-server")
```

Comment:
in my opinion this can be removed, bc `TEST_WEAVIATE_IMAGE` is being set with the `./test/run.sh` script run and if you want to test it solely just by running it with `go test` then testcontainers will build you an image from source on the fly, it's a fast process

Reply:
Agreed, and It will be remove it in the next PR